src/modules/global_css/rsuite.less

```diff
@@ -20,6 +20,10 @@
   width: 800px;
   height: 500px;
 
+  a {
+    text-decoration: none;
```

Comment:
does the underline still appear when you hover links? i think the intended behavior is that links by default do not have underscores, but do upon hover.